salt/elife-xpub/scripts/setup-database.sh

```diff
@@ -6,6 +6,11 @@ DB_CREATED_COMMAND="psql -c \"SELECT 'public.entities'::regclass\""
 DB_ENV="-e PGHOST=${PGHOST} -e PGPORT=${PGPORT} -e PGUSER=${PGUSER} -e PGDATABASE=${PGDATABASE} -e PGPASSWORD=${PGPASSWORD}"
 SETUP_ARGS="--username={{ pillar.elife_xpub.database.user }} --password={{ pillar.elife_xpub.database.password }} --email={{ pillar.elife_xpub.database.email }}"
 
+if [ ! -z "${FORCE}" ]; then
```

Comment:
better name? `clobber` seems a violent/unclear metaphor for no reason

Reply:
`force` seems a standard term for actually execute DB changes vs a dry run.

(A little different, but `CLOB` already has meaning in DBs too.)

Reply:
Personally - I'd prefer pubsweet to have used a different command argument like `dropdb` - rather than calling `setupdb` with `--clobber``